src/headless-cms/components/CmsPageSearch/CmsPageSearch.tsx

```diff
@@ -0,0 +1,179 @@
+import { Card, SearchInput } from 'hds-react';
+import Link from 'next/link';
+// import { useRouter } from 'next/router';
+import React from 'react';
+
+import { ROUTES } from '../../../domain/app/routes/constants';
+import { getEventPlaceholderImage } from '../../../domain/event/utils';
+import {
+  Page,
+  PageIdType,
+  usePagesSearchQuery,
+  useSubPagesSearchQuery,
+} from '../../../generated/graphql-cms';
+import { useCMSClient } from '../../cmsApolloContext';
+import styles from './cmspagesearch.module.scss';
+
+const CmsPageSearch: React.FC<{
+  page?: Page | undefined | null;
+}> = ({ page }): JSX.Element => {
```

Comment:
Return type no necessarily needed when React.FC is already provided (it already includes it)